--- dr_switch_open.sh	2021-02-03 17:30:54.000000000 +0800
+++ dr_switch.sh	2021-02-03 17:30:52.000000000 +0800
@@ -1,7 +1,7 @@
 #!/bin/bash
-#open
+#close
 
-${TOOL_100_PY} w mac ${CHN} 0x015 0xf
+${TOOL_100_PY} w mac ${CHN} 0x015 0x0
 ${TOOL_100_PY} w mac ${CHN} 0x013 0x10000
 ${TOOL_100_PY} w mac ${CHN} 0x014 0x5
 ${TOOL_100_PY} w mac ${CHN} 0x009 0x1
@@ -20,6 +20,9 @@
 ${TOOL_100_PY} w mac ${CHN} 0x00b 0xf0f00
 ${TOOL_100_PY} w mac ${CHN} 0x00b 0xf0f00
 ${TOOL_100_PY} w mac ${CHN} 0x00b 0xf0f00
+${TOOL_100_PY} w rsfec ${CHN} 0x014 0x0
+${TOOL_100_PY} w rsfec ${CHN} 0x015 0x0
+${TOOL_100_PY} w rsfec ${CHN} 0x005 0x0
 ${TOOL_100_PY} w phy ${CHN} 0 0x214 0x0
 ${TOOL_100_PY} w phy ${CHN} 0 0x215 0x0
 ${TOOL_100_PY} w phy ${CHN} 0 0x216 0x0
@@ -36,10 +39,10 @@
 ${TOOL_100_PY} w phy ${CHN} 0 0x225 0x0
 ${TOOL_100_PY} w phy ${CHN} 0 0x226 0x3
 ${TOOL_100_PY} w phy ${CHN} 0 0x227 0x0
-${TOOL_100_PY} w phy ${CHN} 0 0x4 0xc7
-${TOOL_100_PY} w phy ${CHN} 0 0x5 0x2c
+${TOOL_100_PY} w phy ${CHN} 0 0x4 0xcb
+${TOOL_100_PY} w phy ${CHN} 0 0x5 0x4c
 ${TOOL_100_PY} w phy ${CHN} 0 0x6 0xf
-${TOOL_100_PY} w phy ${CHN} 0 0x7 0x86
+${TOOL_100_PY} w phy ${CHN} 0 0x7 0xa6
 ${TOOL_100_PY} w phy ${CHN} 0 0x30 0x42
 ${TOOL_100_PY} w phy ${CHN} 0 0x31 0x20
 ${TOOL_100_PY} w phy ${CHN} 0 0x32 0x0
@@ -60,10 +63,10 @@
 ${TOOL_100_PY} w phy ${CHN} 1 0x225 0x0
 ${TOOL_100_PY} w phy ${CHN} 1 0x226 0x3
 ${TOOL_100_PY} w phy ${CHN} 1 0x227 0x0
-${TOOL_100_PY} w phy ${CHN} 1 0x4 0xc7
-${TOOL_100_PY} w phy ${CHN} 1 0x5 0x2c
+${TOOL_100_PY} w phy ${CHN} 1 0x4 0xcb
+${TOOL_100_PY} w phy ${CHN} 1 0x5 0x4c
 ${TOOL_100_PY} w phy ${CHN} 1 0x6 0xf
-${TOOL_100_PY} w phy ${CHN} 1 0x7 0x86
+${TOOL_100_PY} w phy ${CHN} 1 0x7 0xa6
 ${TOOL_100_PY} w phy ${CHN} 1 0x30 0x42
 ${TOOL_100_PY} w phy ${CHN} 1 0x31 0x20
 ${TOOL_100_PY} w phy ${CHN} 1 0x32 0x0
@@ -84,10 +87,10 @@
 ${TOOL_100_PY} w phy ${CHN} 2 0x225 0x0
 ${TOOL_100_PY} w phy ${CHN} 2 0x226 0x3
 ${TOOL_100_PY} w phy ${CHN} 2 0x227 0x0
-${TOOL_100_PY} w phy ${CHN} 2 0x4 0xc7
-${TOOL_100_PY} w phy ${CHN} 2 0x5 0x2c
+${TOOL_100_PY} w phy ${CHN} 2 0x4 0xcb
+${TOOL_100_PY} w phy ${CHN} 2 0x5 0x4c
 ${TOOL_100_PY} w phy ${CHN} 2 0x6 0xf
-${TOOL_100_PY} w phy ${CHN} 2 0x7 0x86
+${TOOL_100_PY} w phy ${CHN} 2 0x7 0xa6
 ${TOOL_100_PY} w phy ${CHN} 2 0x30 0x42
 ${TOOL_100_PY} w phy ${CHN} 2 0x31 0x20
 ${TOOL_100_PY} w phy ${CHN} 2 0x32 0x0
@@ -108,26 +111,18 @@
 ${TOOL_100_PY} w phy ${CHN} 3 0x225 0x0
 ${TOOL_100_PY} w phy ${CHN} 3 0x226 0x3
 ${TOOL_100_PY} w phy ${CHN} 3 0x227 0x0
-${TOOL_100_PY} w phy ${CHN} 3 0x4 0xc7
-${TOOL_100_PY} w phy ${CHN} 3 0x5 0x2c
+${TOOL_100_PY} w phy ${CHN} 3 0x4 0xcb
+${TOOL_100_PY} w phy ${CHN} 3 0x5 0x4c
 ${TOOL_100_PY} w phy ${CHN} 3 0x6 0xf
-${TOOL_100_PY} w phy ${CHN} 3 0x7 0x86
+${TOOL_100_PY} w phy ${CHN} 3 0x7 0xa6
 ${TOOL_100_PY} w phy ${CHN} 3 0x30 0x42
 ${TOOL_100_PY} w phy ${CHN} 3 0x31 0x20
 ${TOOL_100_PY} w phy ${CHN} 3 0x32 0x0
 ${TOOL_100_PY} w phy ${CHN} 3 0x33 0x0
 ${TOOL_100_PY} w mac ${CHN} 0x00d 0x0
-${TOOL_100_PY} w mac ${CHN} 0x00d 0xff010000
-${TOOL_100_PY} w mac ${CHN} 0x30e 0x197
+${TOOL_100_PY} w mac ${CHN} 0x00d 0xff000100
+${TOOL_100_PY} w mac ${CHN} 0x30e 0x39f
 ${TOOL_100_PY} w mac ${CHN} 0x40b 0x9ffd8028
-${TOOL_100_PY} w rsfec ${CHN} 0x004 0x0
-${TOOL_100_PY} w rsfec ${CHN} 0x005 0xf
-${TOOL_100_PY} w rsfec ${CHN} 0x013 0x0
-${TOOL_100_PY} w rsfec ${CHN} 0x030 0x0
-${TOOL_100_PY} w rsfec ${CHN} 0x010 0x0
-${TOOL_100_PY} w rsfec ${CHN} 0x011 0x0
-${TOOL_100_PY} w rsfec ${CHN} 0x014 0x11
-${TOOL_100_PY} w rsfec ${CHN} 0x015 0x11
 ${TOOL_100_PY} w mac ${CHN} 0x00c 0x0
 ${TOOL_100_PY} w mac ${CHN} 0x00e 0x20
 ${TOOL_100_PY} w mac ${CHN} 0x00e 0x21
